lib/core/utils/custom_toast.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:sufi_ishq/core/utils/constant.dart';
+
+class CustomToast {
```

Comment:
Rename this class to `AppToast` or something related to app.

Reply:
done